src/server/db/seed.ts

```
@@ -180,6 +170,15 @@ const seed = async () => {
 
   /** seed the tags table */
   await db.execute(sql`TRUNCATE TABLE sanbi_tags CASCADE`);
+  const seedTags: NewTag[] = [
+    { tag: "God's love", organizationId: organization!.id },
+    { tag: "God's sovereignty", organizationId: organization!.id },
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)